vertica_python/vertica/connection.py

```diff
@@ -292,8 +292,9 @@ def startup_connection(self):
         user = self.options['user'].encode(ASCII)
         database = self.options['database'].encode(ASCII)
         password = self.options['password'].encode(ASCII)
+        label = self.options['label'].encode(ASCII) if 'label' in self.options else None
 
-        self.write(messages.Startup(user, database))
+        self.write(messages.Startup(user, database, label))
```

Comment:
If we use `options` parameter, it would be something like
```python
options = None
if 'label' in self.options:
    options['label'] = self.options['label'].encode(ASCII)
self.write(messages.Startup(user, database, options)) 
```